core/spec/models/spree/promotion_handler/shipping_spec.rb

```diff
@@ -73,6 +73,8 @@ module PromotionHandler
       end
 
       context 'with a custom shipping action' do
+        let!(:default_actions) { Spree::Config.environment.promotions.shipping_actions }
```

Comment:
Is it not possible to stub this one as well?

Reply:
👍 yes it is, thanks for noticing it!